app/Http/Middleware/LoginVerification.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use App\Models\LoginCredential;
+
+class LoginVerification
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        if(!is_null(session('login_credential_token')) && LoginCredential::where('token', session('login_credential_token'))->exists()){
+            $request->session()->put('login_credential_token', $request->session()->get('login_credential_token'));
+        }else{
+            return to_route('login_credential.create');
+        }
```

Comment:
```suggestion
        $login_credential_token = session('login_credential_token');

        if(is_null($login_credential_token) || !LoginCredential::where('token', $login_credential_token)->exists()){
            return to_route('login_credential.create');
        }
        
        $request->session()->put('login_credential_token', $login_credential_token);
```
このロジックの方が読みやすいと思います。
また値を取得する処理が長い場合は変数を定義して使いまわす方が更に読みやすくなります。
変数を定義する・しないで迷うと思いますが、状況に応じて何を優先するか考えてください。

Reply:
ご指摘、ご説明ありがとうございます。
値を取得する処理が長いものは変数に定義していこうと思います。
修正致しましたので、ご確認よろしくお願い致します。

app/Http/Middleware/LoginVerification.php